### ISSUE TITLE
Table of context sidebar does not work properly with collapsable sections (Wikipedia)

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-android#309]
The main content headers are clickable. Clicking on the sub-headers has no effect unless the associated content sections are already expanded. I think that clicking the header titles should also expand the appropriate content sections if needed

**Comment by kelson42** [comment]
Probably caused by https://github.com/openzim/mwoffliner/issues/136

**Comment by kelson42** [comment]
This is a problem which should be fixed with the ZIM JS Api, we should provide a method "gotosection"

**Comment by macgills** [comment]
@kelson42 is there an issue on some other project to add this API for us to consume?

**Comment by kelson42** [comment]
@macgills Yes, this is here https://github.com/openzim/mwoffliner/issues/596. Of course this is not dedicated to MWoffliner it will be part of ZIM format specs and of course we will have default implementations. But we will start with MWoffliner. This will definitly happens in the next months now.

**Comment by macgills** [comment]
I would say this bug is not getting fixed for 3.4, moving to 3.5

**Comment by kelson42** [comment]
We don't have collapsible section for Mediawiki anymore, so I guess we have not this problem anymore although theoritically it could happen again.
